inyoka_theme_default/templates/portal/privmsg/index.html

```diff
@@ -0,0 +1,209 @@
+{#
+  portal/privmsg/index.html
+  ~~~~~~~~~~~~~~~~~~~~~~~~~
+
+  Display a list of private messages.
+
+  :copyright: (c) 2007-2015 by the Inyoka Team, see AUTHORS for more details.
+  :license: BSD, see LICENSE for more details.
+#}
+
+{% import 'macros.html' as macros %}
+
+{% extends 'portal/privmsg/base.html' %}
+
+{% block breadcrumb %}
+  {{ macros.breadcrumb_item(_('Portal'), href('portal', 'privmsg', 'inbox')) }}
+{% endblock %}
+
+{% block content %}
+  {%- if message %}
+    <div class="message">
+      <div class="head">
+        <div class="row">
+          <div class="col-sm-9">
+            <dl>
+              <dt>{% trans %}From:{% endtrans %}</dt>
+              <dd>
+                <a{% if not message.author.is_active %} class="user_inactive"{% endif %} href="{{ message.author|url }}">
+                  {{- message.author|e }}
+                </a>
+              </dd>
+              <dt>{% trans %}To:{% endtrans %}</dt>
+              <dd>
+                {% for user in message.recipients -%}
+                  <a{% if not user.is_active %} class="user_inactive"{% endif %} href="{{ user|url }}">{{ user|e }}</a>
+                  {%- if not loop.last %}, {% endif %}
+                {% else %}
+                  …
+                {% endfor %}
+              </dd>
+              <dt>{% trans %}Subject:{% endtrans %}</dt>
+              <dd>{{ message.subject|e }}</dd>
+              <dt>{% trans %}Date:{% endtrans %}</dt>
+              <dd>{{ message.pub_date|datetime }}</dd>
+            </dl>
+          </div>
+          <div class="col-sm-3 avatar">{{ macros.add_user_avatar(message.author) }}</div>
+        </div>
+      </div>      
+      <div class="body">
+        {{ message.rendered_text }}
+      </div>
+      <div class="foot">
```

Comment:
Spontaneous Idea: use list-inline instead of the |-seperator. Should be easy as:

    &:extend(.list-inline);
    li {
      &:extend(.list-inline > li);
    }